### PR TITLE
Allow disable group in preview

### PR DESCRIPTION
**Pull request by edoshor**

### CODE REVIEW

File: src/apps/ShidurApp/ShidurToran.js [hunk 1]
```diff
@@ -30,22 +30,29 @@ class ShidurToran extends Component {
         this.props.setProps({group});
     };
 
-    closePopup = () => {
+    closePopup = ({disable=false}={}) => {
+        if (disable) {
+            this.disableRoom(this.state.group);
```
Comment (amnonbb) [inline src/apps/ShidurApp/ShidurToran.js]: Here group have to be taken from props.

File: src/apps/ShidurApp/ShidurToran.js [hunk 2]
```diff
@@ -30,22 +30,29 @@ class ShidurToran extends Component {
...
-            let group = disabled_rooms.find(r => r.room === data.room);
-            if(group) return;
-            disabled_rooms.push(data);
-            this.props.setProps({disabled_rooms});
-            this.props.gerGroups();
+            this.disableRoom(data);
         }
     };
 
+    disableRoom(data) {
```
Comment (amnonbb) [inline src/apps/ShidurApp/ShidurToran.js]: Why we does not use **disableRoom = (data) =>** sysntax here?

File: src/apps/ShidurApp/ShidurToran.js (hunk 1 above)
Comment (edoshor) [reply src/apps/ShidurApp/ShidurToran.js]: done

File: src/apps/ShidurApp/ShidurToran.js (hunk 2 above)
Comment (edoshor) [reply src/apps/ShidurApp/ShidurToran.js]: fixed

**Pull request merged**